server/proof_fetcher.go

```diff
@@ -24,6 +24,9 @@ import (
 	"github.com/google/trillian/storage"
 )
 
+// proofMaxBitLen is the max depth of a tree. Used for storage.NodeID creation.
+const proofMaxBitLen = 64
```

Comment:
Worth being explicit in the const name and comment that this is for logs only?

Reply:
Might be worth renaming these files to `log_proof_...` etc. and maybe prefixing the methods with `log` too?

Fine for a follow-up if you agree in principle.

Reply:
Yep, let's do in in another PR.